src/pages/UI/index.tsx

```diff
@@ -1,7 +1,14 @@
 import { FC } from 'react'
-import { PlusSquare } from 'react-feather'
+import {
+  CheckCircle,
+  Loader,
+  PieChart,
+  PlusSquare,
+  Settings,
+} from 'react-feather'
 
 import { Button, Typography } from '@/components'
+import { Tab } from '@/components/Tab'
```

Comment:
I suggest we follow the convention of importing from `@/components` here too.

In `@/components/index.ts`:

```ts
export { Tab } from './Tab'
```

```suggestion
import { Button, Tab, Typography } from '@/components'
```